README.md

```diff
@@ -54,11 +54,31 @@ Here's how to contribute to Exercises:
 Development Quick Start
 -----------------------
 
-### Use swerve
+### Use Docker
 
-We provide a virtual environment that you can use for development. It has everything you need to get going.
+```bash
+$> docker-compose up
+```
+
+Or use `docker-compose up -d` for a daemonized run.
+
+Drop into a bash shell with
+
+```bash
+$> docker-compose run api bash
+```
 
-See [OpenStax Swerve](http://github.com/openstax/swerve) for more information.
+To run tests, make sure the test database is ready:
+
+```bash
+$> docker-compose run api bundle exec rake db:test:prepare
+```
+
+Then
+
+```bash
+$> docker-compose run api bundle exec rspec
+```
 
```

Comment:
Since we're moving toward using docker for dev tasks, should we add a standard set of rake commands?  `rake shell` maybe?